hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/debug/TestLDBCli.java

```diff
@@ -19,6 +19,8 @@
 import com.fasterxml.jackson.core.type.TypeReference;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.ObjectWriter;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
```

Comment:
We want to use jackson instead of gson due to some issues with gson in java17+. Refer this jira: https://issues.apache.org/jira/browse/HDDS-10538.
Could you please implement the logic with jackson instead?

Reply:
Thanks @Tejaskriya  for the comment and view.
I have updated it.